core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1072,6 +1072,15 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val SHUFFLE_FETCH_CONTINUOUS_BLOCKS_IN_BATCH =
+    ConfigBuilder("spark.shuffle.fetchContinuousBlocksInBatch")
```

Comment:
is it possible to make it a SQLConf? e.g. we can read the conf in `ShuffledRowRDD` and set a special flag in `TaskContext` local properties.

Then it's easy to switch this config at runtime.

Reply:
Thanks, done in fa7c272.